### PR TITLE
Add documentation for new input_select functionality

### PR DESCRIPTION
**Pull request by l-mb**
Accompanies https://github.com/home-assistant/core/pull/42968

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/input_select.markdown [hunk 1]
```diff
@@ -80,10 +80,26 @@ This integration provides three services to modify the state of the `input_selec
 | ------- | ---- | ----------- |
 | `select_option` | `option` | This can be used to select a specific option.
 | `set_options` | `options`<br>`entity_id(s)` | Set the options for specific `input_select` entities.
-| `select_previous` | | Select the previous option.
-| `select_next` | | Select the next option.
+| `select_first` | | Select the first option.
+| `select_last` | | Select the last option.
 | `reload` | | Reload `input_select` configuration |
 
+#### Service `input_select.next`
```
Comment (frenck) [inline source/_integrations/input_select.markdown]:
```suggestion
#### Service `input_select.select_next`
```

File: source/_integrations/input_select.markdown [hunk 2]
```diff
@@ -80,10 +80,26 @@ This integration provides three services to modify the state of the `input_selec
...
+
+Select the next option.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `cycle` | yes | Whether to cycle to the first value after the last. Default: `true`
+
+#### Service `input_select.previous`
```
Comment (frenck) [inline source/_integrations/input_select.markdown]:
```suggestion
#### Service `input_select.select_previous`
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @l-mb 👍 

Left a small comment. Could you take a look?

File: source/_integrations/input_select.markdown (hunk 2 above)
Comment (l-mb) [reply source/_integrations/input_select.markdown]: Oops! Too many `select`s ;-) Fixed and merged.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**